src/main.rs

```diff
@@ -45,7 +45,7 @@ fn main() {
                                .long("output-type")
                                .value_name("OUTPUT TYPE")
                                .default_value("lcov")
-                               .possible_values(&["ade", "lcov", "coveralls", "coveralls+", "files"])
+                               .possible_values(&["ade", "lcov", "coveralls", "coveralls+", "files", "fuzzmanager"])
```

Comment:
Maybe we could find a nicer name for the format, something that says it's a recursive coverage format.

Reply:
Yes maybe... :)
I've absolutely no idea

Reply:
recursive_coverage
recoverage
recover
recov

Do you like any of this?

Reply:
recov is nice.

We could also imply it's *our own* format: `mozcov`,  `mozilla_coverage`,

Reply:
or covcov :)

Reply:
+1 for mozcov

Reply:
I like any of recov, mozcov, covcov (hint to bugbug :P).